packages/create-plugin/src/utils/utils.config.ts

```diff
@@ -84,9 +84,17 @@ function readRCFileSync(path: string): CreatePluginConfig | undefined {
 }
 
 function createFeatureFlags(flags?: FeatureFlags): FeatureFlags {
-  // Default values for new scaffoldings
+  // For new scaffolds override any defaults with args passed in the CLI.
   if (commandName === 'generate') {
-    return DEFAULT_FEATURE_FLAGS;
+    const flags = Object.entries(DEFAULT_FEATURE_FLAGS).reduce((acc, [flag, value]) => {
+      if (argv.hasOwnProperty(flag)) {
+        return { ...acc, [flag]: argv[flag] };
+      } else {
+        return { ...acc, [flag]: value };
+      }
+    }, {} as FeatureFlags);
+
+    return flags;
```

Comment:
Would it make sense to also allow CLI feature flags for the other commands (and not just generate)?

(I think what I would expect is to be able to use these flags the same way when updating plugins. It could also be useful for writing migration guides, as we could share a simple code-snippet instead of explaining how to edit the `.cprc.json` - as you pointed it out in the PR description, too.)

Reply:
Have done as you suggested in https://github.com/grafana/plugin-tools/pull/984/commits/9ef654f26eeb66e951aa4cd2cacecbf632295c1e @leventebalogh ! 👍